### ISSUE TITLE
Does the react-native-iap support localised title?

### ISSUE THREAD

**Issue by cervebar** [hyochan/react-native-iap#2458]
**Description**

We have localized iap products (very likely) correctly defined in Google Play and in App Store. In code we are using change language feature (via `i18n`) is there anyhow possible to retrieve localised title/name of products?

**Expected**
someting like:
```
const products = await getProducts({skus: productIds})
const locTitle = products[0].localisedTitle
```

**Environment:**

- react-native-iap: any version
- react-native: `0.68`
- Platforms (iOS, Android, emulator, simulator, device): iOS, Android

**To Reproduce**

```
const products = await getProducts({skus: productIds})
console.log(products)
```

---

**Comment by professorkolik** [comment]
As far as I can say it will take languages which are set to user AppStore/PlayStore

Based on docs Product contains

title => Returns the title Android and localizedTitle on iOS.
description => Returns the localized description on Android and iOS.

If you wanna show based on user settings localization, then you should handle it on your end

**Comment by MTPL0005-AbhishekDubey** [comment]
Yes, it returns the localized price for iOS and Android, but the key that produces the localized price differs.
for iOS --> sub.localizedPrice will give the localized price,
for Android --> sub.subscriptionOfferDetails[0].pricingPhases.pricingPhaseList[0].formattedPrice

here "sub" stores the data returned by IAP.getSubscriptions

**Comment by hyochan** [comment]
I’m closing all issues reported in versions below 14, as the library now supports the new architecture with NitroModules and has been completely revamped.
